### ISSUE TITLE
[Chrome] Changing frame's opacity blurs the header text (also changes letter spacing?)

### ISSUE THREAD

**Issue by MitjaBezensek** [tldraw/tldraw#1993]
Probably a Chrome rendering issue?

https://github.com/tldraw/tldraw/assets/2523721/a836f04d-dcc7-4ca6-a5de-448a6d32315a

**Comment by mimecuvalo** [comment]
I don't see this afaict - @MitjaBezensek reopen though if you're still seeing this!

**Comment by MitjaBezensek** [comment]
Yes, I still see it. Seems to only happen on my external monitor, so it might have something to do with non-retina screens.

**Comment by mimecuvalo** [comment]
Huh weird, but why only for frame header text??
